docs/documentation/features.md

```diff
@@ -1,52 +0,0 @@
----
-title: Features of Saros
```

Comment:
This page still provided a good overview of the Saros featuer set, which is missing from the new page layout. What are your plans to address this?

Reply:
The initial idea was to have all features on the landing page, but I think this leads to a crowded page. I will recreate the file.

Reply:
How about opening an issue to merge the feature and screenshot page and link the complete awareness information section in the "Awareness Information" section of the landing page?